### PR TITLE
minor text improvements

### PR DESCRIPTION
**Pull request by catcad**
Fix #Issue

**Changes proposed in this pull request**:
- minor text updates

### CODE REVIEW

**Review by TheOneAndra — APPROVED** [review]

File: website/templates/index.html [hunk 1]
```diff
@@ -46,14 +46,14 @@ <h2>Energy cells explained</h2>
             <p>The fundamental idea of the cellular approach is to balance energy consumption and energy production at the smallest level possible</p>
```
Comment (TheOneAndra) [inline website/templates/index.html]: I suggest this change here as it seems to be more in line with the "selling points" descibed lower for open_plan.
(+It seems that not everyone agrees on this balancing of consumption and generation in an energy cell)

```suggestion
            <p>The fundamental idea of the cellular approach is to optimize energy system planning at all possible levels, from the smallest energy cell to the largest one.</p>
```

Comment (catcad) [reply website/templates/index.html]: I agree to your suggestion! Shall I commit it accordingly by clickling on commit suggestion? --> I am not sure about the correct GitHub Etiqutte ;)

Comment (Bachibouzouk) [reply website/templates/index.html]: yes :)

Comment (Bachibouzouk) [reply website/templates/index.html]: Then you can click "merge" since Andra approved your pull request :)

**Comment by Bachibouzouk** [comment]
perfect @catcad, was it your first merge?

Once the branch is merged, we don't need it anymore, you can click on the "Delete branch" button

**Comment by catcad** [comment]
> 
> 
> perfect @catcad, was it your first merge?
> 
> Once the branch is merged, we don't need it anymore, you can click on the "Delete branch" button

Yes, thanks for the hint with deleting - makes lots of sense :)

**Pull request merged**